bedrock/mozorg/templates/mozorg/about/governance/policies/participation.html

```diff
@@ -11,8 +11,10 @@
 
 
 {% block article %}
-<h1 class="mzp-c-article-title">{{ ftl('participation-mozilla-community') }}</h1>
-<h4>{{ ftl('participation-version-31-updated') }}</h4>
+<header>
+  <h1 class="mzp-c-article-title">{{ ftl('participation-mozilla-community') }}</h1>
+  <p><strong>{{ ftl('participation-version-31-updated') }}</strong></p>
```

Comment:
**design (non-blocking)**: I kind of like the PR sizing, but want to note this is a discrepancy from the prod H4 sizing, so we might need to add a class if we want parity

prod
<img width="654" alt="Screenshot 2024-12-17 at 10 09 44 AM" src="https://github.com/user-attachments/assets/6283639f-cc77-4608-90e6-ce60a4a43906" />

pr
<img width="710" alt="Screenshot 2024-12-17 at 10 09 49 AM" src="https://github.com/user-attachments/assets/d62658d7-a590-41fb-8476-398058a7e984" />

Reply:
I was kind of trying to resist making a custom style for just one headline, but you convinced me to try and keep things consistent 👍